lib/game.rb

```diff
@@ -0,0 +1,162 @@
+#game class file
+require './lib/cell'
+require './lib/board'
+require './lib/ship'
+
+ class Game
+
+  attr_reader :player, :cruiser, :submarine
+  attr_accessor :computer_board
+
+  def initialize(player)
+    @player = player
+    @computer_board = Board.new
+    @player_board = Board.new
+    @cruiser = Ship.new("Cruiser", 3)
+    @submarine = Ship.new("Submarine", 2)
+
+  end
+
+  def user_places_ships(ship, coordinates)
+    loop do
+      p "which ship would you like to place? Choose either cruiser or submarine. q to quit"
+      ship = gets.chomp.downcase
+
+        if ship == "q"
+          break
+        elsif ship == cruiser.name.downcase
+          ship_var = cruiser
+        elsif ship == submarine.name.downcase
+          ship_var = submarine
+        elsif ship != cruiser.name.downcase || ship != submarine.name.downcase
+          puts "Please enter a valid ship!"
+        end
+
```

Comment:
There has to be a better way to convert the users input into an array. The way it works now is the user inputs `A1 A2 A3` without commas or quotes and it will work. If any other characters get added in it will cause an error.

Reply:
There are definitely methods you can use on user input to strip it of extra spaces and punctuation. Have you tried those?

Reply:
I tried a bunch but couldnt get them to work. I think it has something to do with `gets.chomp` but im not sure. I figured this would work for the time being.

Reply:
got it working in my new PR!